crypto/params.c

```diff
@@ -1529,3 +1529,29 @@ int OSSL_PARAM_get_octet_string_ptr(const OSSL_PARAM *p, const void **val,
     return rv || get_string_ptr_internal(p, val, used_len,
                                          OSSL_PARAM_OCTET_STRING);
 }
+
+OSSL_PARAM *OSSL_PARAM_next(OSSL_PARAM *p)
```

Comment:
I don't think this and the next one should be functions.  The replace `p++` with `p = OSSL_PARAM_next(p);`